src/Stratis.Bitcoin.Features.WatchOnlyWallet/WatchOnlyWallet.cs

```diff
@@ -162,11 +162,17 @@ public class TransactionData
         [JsonProperty(PropertyName = "hex")]
         public string Hex { get; set; }
 
+        /// <summary>
+        /// Allow <see cref="Transaction" /> to be used with networks other than Main.
+        /// </summary>
+        [JsonIgnore]
+        public Network Network { get; set; }
+
         /// <summary>
         /// A transaction affecting a script being watched.
         /// </summary>
         [JsonIgnore]
-        public Transaction Transaction => Transaction.Parse(this.Hex);
+        public Transaction Transaction => Transaction.Load(this.Hex, this.Network ?? Network.Main);
```

Comment:
I think we should delete this property, it only complicates stuff.

Reply:
Ok the property is gone now. Added some Transaction.Loads's elsewhere in the code.

Also removed the serialization settings based on the fact that the TransactionData.Id is equal to the transaction hash which means we don't need to load the transaction within ReadJson to get at the hash.